apps/chef_db/src/chef_sql.erl

```diff
@@ -114,15 +105,7 @@
 -include_lib("sqerl/include/sqerl.hrl").
 -include("../../include/chef_types.hrl").
 
--type delete_query() :: delete_cookbook_version_by_id |
-                        delete_user_by_id |
-                        delete_data_bag_by_id |
-                        delete_data_bag_item_by_id |
-                        delete_environment_by_id |
-                        delete_client_by_id |
-                        delete_node_by_id |
-                        delete_role_by_id |
-                        delete_sandbox_by_id.
+-type delete_query() :: atom().
```

Comment:
I would rather have it specific at the source if we want to spec it to that depth  (at least  chef_object:delete_query) than have to maintain a separate list of possible returns in a different app, since those returns are owned by the chef_object app and not chef_sql.